browser/src/control/Control.FormulaBarJSDialog.js

```diff
@@ -254,10 +256,12 @@ class FormulaBar {
 
 	focusField() {
 		L.DomUtil.addClass(this.getInputField(), 'focused');
+		app.preventExit();
```

Comment:
So if I got it right exiting is prevented even if the user didn't modify the formula bar content ?